CorinneTest/Person.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+namespace CorinneTest
+{
+    public class Person 
+    {
+        private string name;
+        public string Name
+        {
+            get => this.name; 
+            set { this.name = value; }
+        }
+
+        private int? age;
```

Comment:
side note, you could completely remove int age and always return a computed value in get{} only from birthDate and have no set.